server/config/passport.js

```diff
@@ -0,0 +1,25 @@
+const JwtStrategy = require("passport-jwt").Strategy;
+const { ExtractJwt } = require("passport-jwt");
+
+const { passportSecret } = require("./keys");
+const User = require("../models/User");
+
+const options = {
+  jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+  secretOrKey: passportSecret,
+};
+
+module.exports = (passport) => {
+  passport.use(
+    new JwtStrategy(options, (jwtPayload, done) => {
+      User.findById(jwtPayload.id, (err, user) => {
+        if (err) return done(err, false);
+
+        if (user) {
+          return done(null, user);
```

Comment:
perfect - now the team should know @gersonesp @stanley-c-so  - that if you are using this middleware, you have access to the user using req.user 

I don't want to see another fetch of the user when this middleware is being used